frontend/src/router/routeHandlers.ts

```diff
@@ -71,9 +71,17 @@ const beforeEnterGrantUserPermissionRoute = async (
     return true;
 };
 
+const beforeEnterGrantApplicationAdminRoute = async (
+    to: RouteLocationNormalized
+) => {
+    populateBreadcrumb([routeItems.dashboard, routeItems.grantAppAdmin]);
```

Comment:
I think before enter this router, we have to check the selected application is FAM. Cause only FAM admin is allowed to go to this route. 

In the below example, I selected FOM_DEV, but I go to the `/grant-app-admin` router directly, it still lets me in, it's super confused. So for this ticket, let's just restrict only allow enter this router when the selected application is FAM. 
 <img width="1042" alt="Screen Shot 2024-01-31 at 2 05 29 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/1ee4b6ba-fae1-437f-847c-82d83dc1ecf1">
<img width="998" alt="Screen Shot 2024-01-31 at 2 06 18 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/e733298b-9525-443d-8df4-8a14358f2c4e">

Reply:
I agree with you. I haven't thought about it, my mistake. I'll update it to restrict access.

Reply:
Most of the routes should have checks/guards to protect at very beginning. But just to inform, the guards (including this one) on frontend will be shift on its logic soon due to our new access change for users.